pac4j-saml/src/test/java/org/pac4j/saml/client/SAML2ProfileTest.java

```diff
@@ -0,0 +1,51 @@
+package org.pac4j.saml.client;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.pac4j.core.context.Pac4jConstants;
+import org.pac4j.core.profile.Gender;
+import org.pac4j.saml.profile.SAML2Profile;
+
+/**
+ * Generic test for {@link SAML2Profile}
+ */
+public class SAML2ProfileTest {
```

Comment:
Can you rename the class with `Tests` at the end? Otherwise, the test won't be played at build: https://github.com/pac4j/pac4j/blob/master/pom.xml#L193